src/silo/database.cpp

```diff
@@ -172,9 +193,9 @@ BitmapContainerSize::BitmapContainerSize(size_t genome_length, size_t section_le
       total_bitmap_size_computed(0) {
    size_per_genome_symbol_and_section["NOT_N_NOT_GAP"] =
       std::vector<size_t>((genome_length / section_length) + 1, 0);
-   size_per_genome_symbol_and_section[genomeSymbolRepresentation(NUCLEOTIDE_SYMBOL::GAP)] =
+   size_per_genome_symbol_and_section["-"] =
```

Comment:
I think we use `"-"` and `"N"` also below (line 326 ff). Could we make this a constant, so we do not change this here by chance and not below?

Reply:
This was the only usage of the helper function genomeSymbolRepresentation which put an additional header include into `nucleotide_symbols`, but we can call the function to return the char and then call an explicit string instructor. Although, this is only user-facing analysis output which is bound to change with #157 anyways. I am fine with both anyways